gammapy/data/observations.py

```diff
@@ -715,6 +715,22 @@ def group_by_label(self, labels):
             obs_groups[f"group_{label}"] = observations
         return obs_groups
 
+    @classmethod
+    def concatenate(cls, observations_list):
+        """Create a new `Observations` instance by concatenating a list of `Observations` objects.
+
+        Parameters
+        ----------
+        observations_list : list of `~gammapy.data.Observations`
+            The list of `Observations` to concatenate.
+
+        Returns
+        -------
+        observations : `~gammapy.data.Observations`
+            The `Observations` object resulting from the concatenation of all the `Observations` in `observation_list`.
+        """
+        return cls(list(np.concatenate(observations_list)))
```

Comment:
Should we not add a test function??
if an obs_id is common, check if the metadata are different. Unfortunately there are no meta data in observation, e.g. DST_VER, ANA_VER, CAL_VER, EVENT_CLASS, EVENT_TYPE, INSTRUME

Reply:
What is the advantage over `observations.concatenate(observations_extra)` ?